packages/gestalt/dist/index.d.ts

```diff
@@ -933,6 +933,22 @@ interface ButtonProps {
   type?: 'button' | 'submit' | undefined;
 }
 
+interface ButtonToggleProps {
+  accessibilityControls?: string | undefined;
+  accessibilityLabel?: string | undefined;
+  color?:
+    | 'red'
+    | 'transparent'
+    | undefined;
```

Comment:
let's remove all the | undefined... they are incorrect indeed, everywhere...

Reply:
done